Standards/scs-0005-v1-project-governance.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Sovereign Cloud Standards
+type: Procedural
+status: Draft
+track: Global
+stabilized_at:
+description: |
+  SCS-0005 outlines the structure and governance of the SCS community by the SCS Project Board and how this is elected.
+---
+
+## Introduction
+
+The [Sovereign Cloud Stack (SCS)](https://scs.community) provides standards
+for a range of cloud infrastructure types as well as a modular open-source
+reference implementation.
+The project is governed by the _SCS Project Board_.
+
+## Role of the _SCS Project Board_
+
+The role of the _SCS Project Board_ is the overall governance of the SCS Community and Project.
+This happens together with the _Forum SCS-Standards_ of the Open Source Business Alliance. To further
+underline this alignment, the _Forum SCS-Standards_ is part of the _SCS Project Board_.
+The _SCS Project Board_ itself is elected by the _SCS Community_.
+
+### Definitions
+
+#### _SCS Project_
+
+The _SCS Project_ is the Open-Source project that consists of the software, documentation, documents, blog posts as well as the people ("_SCS Community_") working on this.
+
+#### _SCS Community_
+
+The collective of people, companies, and organizations promoting the idea of the _SCS Project_ as well as the people working on the various aspects.
+
+## Election of the _SCS Project Board_
+
+### Term
+
+The _SCS Project Board_ is elected for the term of one year. Elections are done
+in November. The December serves as a transition period.
```

Comment:
> Should we add here "starting with November 2025"? Since November 2024 is already over and we still need to do a voting this year?

I addressed this with a better wording (I think ;)